src/Algorithms/threshold/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Define an image to start from 
+#FROM bethcimini/docker-gradio-demo:latest
+FROM gradio-base:latest
+
+# Set the working directory within the container
+WORKDIR /workspace
+
+# Install the dependencies for the gradio app
+RUN python -m pip install scipy==1.9.1 scikit-image==0.20.0
+
+# Add app.py from the directory where `docker build` was run and add it into the
+# workspace directory within the container
+ADD /Algorithms /workspace/
```

Comment:
I don't think we want to add the entire `Algorithms` folder, just `Algorithms/threshold` or just the generated `app.py` if we want to do the generation outside of the docker.

Reply:
Got it. Hence, I am now only adding `config.yml` file and `algorithm-name.py` file (Here, in this case, `threshold.py`)